### PR TITLE
Feature | Update to .Net9.0

### PR DESCRIPTION
**Pull request by arellegue**
This branch was cloned from ErikEJ/issue-2669.

Fixes were added for errors CA2022 and IL2093, so can build locally using msbuild.

### CODE REVIEW

File: tools/props/VersionsNet9OrLater.props [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Net9 project dependencies -->
+  <PropertyGroup>
+    <SystemConfigurationConfigurationManagerVersion>9.0.0-preview.6.24327.7</SystemConfigurationConfigurationManagerVersion>
```
Comment (ErikEJ) [inline tools/props/VersionsNet9OrLater.props:5]: Update to RC1

File: tools/props/VersionsNet9OrLater.props [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    <SystemRuntimeCachingVersion>9.0.0-preview.6.24327.7</SystemRuntimeCachingVersion>
```
Comment (ErikEJ) [inline tools/props/VersionsNet9OrLater.props:6]: Update to RC1

File: tools/props/VersionsNet9OrLater.props [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    <MicrosoftExtensionsCachingMemoryVersion>9.0.0-preview.6.24327.7</MicrosoftExtensionsCachingMemoryVersion>
```
Comment (ErikEJ) [inline tools/props/VersionsNet9OrLater.props:7]: Update to RC1

**Comment by arellegue** [comment]
Closing this PR.

A PR was created that targets ErikEJ's issue-2669 branch, instead.

File: tools/props/VersionsNet9OrLater.props (hunk 3 above)
Comment (arellegue) [reply tools/props/VersionsNet9OrLater.props:7]: Updated to rc1 version.

**Comment by DavoudEshtehari** [comment]
https://github.com/ErikEJ/SqlClient/pull/3

**Pull request closed without merging**